src/index.ts

```diff
@@ -1,9 +1,15 @@
 import { Application } from 'probot' // eslint-disable-line no-unused-vars
 
+const welcomeMessage: string = `Thank you for contributing to Leapp project!
+
+Please note that every PR needs to comply with the [Leapp Guidelines](https://github.com/oamg/leapp-guidelines), pass tests and linter check before it can be merged.
+If you want to re-run tests or request review, you can use following commands as a comment:
+- "leapp-ci build" - runs unit tests
+- "please review" - notifies leapp developers of review request`
```

Comment:
I would also add the 'e2e tests' comment. Although the comment will trigger only for members of oamg team, it's good to have everything in one place (and even oamg members keep asking what the comment is...).

Reply:
Wondering how did 'e2e tests' comment drop out. I think it was there initially...

Reply:
my comments above ;-) we discussed that with Pepa in person as well and decided that drop seems better. But I do not care so much whether there will be the comment or nothing. From my point, we should know this thing everyone and we do not need to spam people. What if we add additional extra functions for us in future? But I am ok with both solutions.

Reply:
I knew I should've put it to cabal :D 
I didn't think this message was necessary for oamg members at all but if people are still struggling with the command then perhaps it makes sense to add e2e. We should do a quick poll or something...